### PR TITLE
Modify image gathering for custom namespaces

### PR DESCRIPTION
**Pull request by maroroman**
## Related Issues and Dependencies

…

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

This PR fixes a bug related to using custom namespaces with the new bugfix introduced in PR #123 

Testing instructions:
1. Deploy the modifed jupyterhub-odh image
2. Attempt spawning notebook in a custom namespace

### CODE REVIEW

File: .jupyter/jupyterhub_config.py [hunk 1]
```diff
@@ -208,10 +208,6 @@ def _options_form_default(self):
   def options_from_form(self, formdata):
     options = {}
     cm_data = self.single_user_profiles.user.get(self.user.name)
-    if custom_notebook_namespace:
-        options['custom_image'] = f'image-registry.openshift-image-registry.svc:5000/{namespace}/%s' % cm_data['last_selected_image']
-    else:
-        options['custom_image'] = cm_data['last_selected_image']
     options['size'] = cm_data['last_selected_size']
     self.gpu_count = cm_data['gpu']
     self.image = options['custom_image']
```
Comment (vpavlin) [inline .jupyter/jupyterhub_config.py:213]: please remove this line. It breaks when the new spawn progress UI is not used

**Comment by vpavlin** [comment]
lgtm

**Review by vpavlin — APPROVED** [review]

**Review by anishasthana — COMMENTED** [review]
/lgtm

**Pull request merged**